src/js/module/main/main.js

```diff
@@ -16,23 +32,42 @@ export default View.extend({
 
   initialize() {
     this.render();
-    this.submit = new Button({
-      modifier: 'finish-session',
-      label: 'zrobione!',
-      visible: false,
-    });
-    this.submit.on('click', this.onClickSubmit, this);
     this.$proxy = this.$el.children();
-    this.$proxy.append(this.submit.$el);
+    this.submit = new Button(SUBMIT_BUTTON_INIT_VALUES);
+    this.submit.on('click', this.onClickSubmit, this);
+
+    const { selectPanoMode } = this.app.options;
```

Comment:
Yep, this looks, that for creating campaing we need to uncomment `// selectPanoMode: true,`